src/components/UploadNoStyle/UploadNoStyle.tsx

```diff
@@ -88,6 +104,34 @@ export const UploadNoStyle = (props: UploadNoStyleProps) => {
     [props, setUploadUrl],
   )
 
+  const _body = () => {
```

Comment:
**issue** defining functions inside components causes rendering bugs. Need to move outside this component's scope, or make it a variable (not a function)

Reply:
Also, why not just put this code directly inline?